.github/workflows/docs.yaml

```diff
@@ -0,0 +1,34 @@
+name: Deploy the documentation
+#on:
+#  release:
+#    types: [created]
```

Comment:
Should we remove this for now? 🤷

Reply:
heh, this is one of the things that was fixed in the bit I broke out of this PR. But yes, will remove it here.